### PR TITLE
web: migrate to web platform- packages

### PR DESCRIPTION
**Pull request by BRKalow**
Migrate away from @hashicorp/nextjs-scripts to the new suite of @hashicorp/platform-* packages

[_Created by Sourcegraph campaign `bkalow/migrate-nextjs-scripts-packages`._](https://sourcegraph.hashi-mktg.com/users/bkalow/campaigns/migrate-nextjs-scripts-packages)

### CODE REVIEW

File: website/components/subnav/index.jsx [hunk 1]
```diff
@@ -27,6 +27,7 @@ export default function ProductSubnav() {
       menuItems={subnavItems}
       constrainWidth
       Link={Link}
+      matchOnBasePath
```
Comment (jescalan) [inline website/components/subnav/index.jsx:30]: Still curious about this one - is there any place it's not needed?

**Review by jescalan — APPROVED** [review]

File: website/pages/downloads/index.jsx [hunk 2]
```diff
@@ -11,7 +11,7 @@ export default function DownloadsPage({ binaryReleases, desktopReleases }) {
   return (
     <div className={styles.root}>
       <HashiHead title={`Downloads | ${productName} by HashiCorp`} />
-      <ProductDownloader
+      <ProductDownloadsPage
```
Comment (jescalan) [inline website/pages/downloads/index.jsx:14]: This needs wider changes - the head section and wrapper are included in the page version of the component

**Review by jescalan — CHANGES_REQUESTED** [review]
Sorry I approved too quick - there are more adjustments needed to change over to the `ProductDownloadsPage` component

File: website/components/subnav/index.jsx (hunk 1 above)
Comment (BRKalow) [reply website/components/subnav/index.jsx:30]: All the subnavs in www don't need it.

File: website/pages/downloads/index.jsx (hunk 2 above)
Comment (BRKalow) [reply website/pages/downloads/index.jsx:14]: Thanks, good looking out! I left out the usage of the server helper as we are also fetching for boundary desktop here in parallel.

Comment (jescalan) [reply website/pages/downloads/index.jsx:14]: Still can have the wrapping div, styles, and extra head component removed, as those are all repeated 😬

Comment (BRKalow) [reply website/pages/downloads/index.jsx:14]: Oh wow I think I have a local commit that I didn't push up 🤦 one moment...

File: website/pages/downloads/index.jsx [hunk 3]
```diff
@@ -1,103 +1,99 @@
...
+        {
+          label: 'Amazon Linux',
+          commands: [
+            'sudo yum install -y yum-utils',
+            'sudo yum-config-manager --add-repo https://rpm.releases.hashicorp.com/AmazonLinux/hashicorp.repo',
+            'sudo yum -y install boundary',
+          ],
+          os: 'linux',
+        },
+      ]}
```
Comment (jescalan) [inline website/pages/downloads/index.jsx:58]: Unless these commands are different than the standard set on purpose, they can be omitted, they are built in to the core

**Review by jescalan — CHANGES_REQUESTED** [review]
Sorry one more small thing with the downloads page

**Review by jescalan — APPROVED** [review]
👏

**Pull request merged**